File: Contest _B/Fair Pen/Same Array.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int t;
    cin >> t;
    
    while(t--)
    {
        int n;
        cin >> n;
        
        vector<int> arr(n);
        for(int &i : arr) cin >> i;
        
        int mx = *max_element(arr.begin(), arr.end());
        for(int &i : arr)
            i += mx;
            
        for(int i : arr)
            cout << i << " ";
        cout << endl;
    }
    return 0;
}

File: Contest _B/Fair Pen/Same Array.py
for _ in range(int(input())):
    n = int(input())
    arr = list(map(int, input().split()))
    
    mx = 0
    for i in range(n):
        if arr[i] > mx:
            mx = arr[i]
            
    for i in range(n):
        arr[i] += mx
        
    print(' '.join(map(str, arr)))